java/tomcat-native/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           java 1.0
 
 name                tomcat-native
-version             1.2.31
+version             1.2.33
 revision            1
```

Comment:
revision should be reset to zero when updating the version

Reply:
Thanks, PR updated.